ksql-common/src/test/java/io/confluent/ksql/GenericRowTest.java

```diff
@@ -122,4 +128,36 @@ public void shouldPrintStructRowCorrectly() {
 
   }
 
-}
\ No newline at end of file
+  @Test
+  public void shouldHandleRowWithNoElements() {
+      List<Object> linkedList = new LinkedList<>();
+      GenericRow genericRow = new GenericRow(linkedList);
+
+      assertThat(genericRow.getColumns().size(), is(0) );
+  }
+
+  @Test
+  public void testEquals(){
+
+      List<Object> columnListWithEmptyObject = new ArrayList<>(1);
```

Comment:
```suggestion
      List<Object> columnListWithEmptyObject = ImmutableList.of(new Object());
```

More importantly, where is this used?